migration/src/m20220101_000001_init.rs

```diff
@@ -124,6 +124,52 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
+        manager
+            .create_table(
+                Table::create()
+                    .table(TokenOwnership::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(TokenOwnership::Id)
+                            .big_integer()
+                            .not_null()
+                            .auto_increment()
+                            .primary_key(),
+                    )
+                    .col(ColumnDef::new(TokenOwnership::Owner).binary().not_null())
+                    .col(ColumnDef::new(TokenOwnership::Mint).binary().not_null())
+                    // TODO: Change this to a u64 here to avoid balance overflow.
+                    .col(
+                        ColumnDef::new(TokenOwnership::Amount)
+                            .big_integer()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(TokenOwnership::Delegate).binary())
+                    .col(ColumnDef::new(TokenOwnership::Frozen).boolean().not_null())
+                    // TODO: Change this to a u64 here to avoid balance overflow.
+                    .col(ColumnDef::new(TokenOwnership::IsNative).big_integer())
+                    // TODO: Change this to a u64 here to avoid balance overflow.
+                    .col(
+                        ColumnDef::new(TokenOwnership::DelegatedAmount)
+                            .big_integer()
+                            .not_null(),
+                    )
+                    .col(ColumnDef::new(TokenOwnership::CloseAuthority).binary())
+                    .to_owned(),
+            )
+            .await?;
+
+        manager
```

Comment:
We also want an owner index (not just owner + mint). its necessary for GetToken[...]ByOwner

Reply:
Yep you are right.